src/python/pants/backend/awslambda/python/awslambda_python_rules_test.py

```diff
@@ -26,9 +25,7 @@ def rule_runner() -> RuleRunner:
     return RuleRunner(
         rules=[
             *awslambda_python_rules(),
-            QueryRule(
-                CreatedAWSLambda, (PythonAwsLambdaFieldSet, OptionsBootstrapper, PantsEnvironment)
-            ),
+            QueryRule(CreatedAWSLambda, (PythonAwsLambdaFieldSet, PantsEnvironment)),
```

Comment:
Should `PantsEnvironment` become a `SessionValue` in a followup?

Reply:
Yes it should!

Reply:
That'll be wonderful to not have to have users need to fiddle with this during test setup. And an API like this would be nice and declarative:

```python
rule_runner.set_env_var("FOO", "val")
```